cloud/linode/loadbalancers.go

```diff
@@ -776,14 +777,25 @@ func makeLoadBalancerStatus(service *v1.Service, nb *linodego.NodeBalancer) *v1.
 		Hostname: *nb.Hostname,
 	}
 	if !getServiceBoolAnnotation(service, annLinodeHostnameOnlyIngress) {
-		ingress.IP = *nb.IPv4
+		if envBoolOptions("LINODE_HOSTNAME_ONLY_INGRESS") {
+			fmt.Printf("LINODE_HOSTNAME_ONLY_INGRESS:  %v\n", envBoolOptions("LINODE_HOSTNAME_ONLY_INGRESS"))
```

Comment:
```suggestion
			fmt.Printf("LINODE_HOSTNAME_ONLY_INGRESS:  %v\n", val)
```

Reply:
Is this for logging purpose? If so, looks like this repo using klog as their logging lib

Example: https://github.com/linode/linode-cloud-controller-manager/blob/master/cloud/linode/loadbalancers.go#L173